### ISSUE TITLE
Workshop Content?

### ISSUE THREAD

**Issue by SgtVal** [Tripperful/moneytrees#1]
Can you please upload the content? Current workshop link is no longer available.

**Comment by DaveJones1** [comment]
Yeah that would be helpful using it for my server

**Comment by Tripperful** [comment]
Yeah I can see the issue... Unfortunately I don't have the source files anymore, but even worse I don't even have the compiled version of the content for this addon.

I will do my best trying to find it somehow and publish it if I succeed.

**Comment by DaveJones1** [comment]
Have you managed to find it?
